examples/src/components/BucketExecutorExample.jsx

```diff
@@ -0,0 +1,250 @@
+                        flexDirection: "column",
+                        alignItems: "flex-start",
+                        minHeight: "3em",
+                        lineHeight: "1em",
+                        margin: "10px 0",
+                        fontSize: 30,
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result.headerItems[0][0].map(header => header.attributeHeaderItem.name).join(", ")}
+                    <br />
+                    {result.data[0][0]}
+                    <div>
+                        <button
+                            disabled={isFirstPage}
+                            onClick={() => getPage({ offset: prevOffset, limit: [1, 1] })}
+                            className={`button button-action s-retry-button ${isFirstPage ? "disabled" : ""}`}
+                        >
+                            Previous
+                        </button>
+                        <button
+                            disabled={isLastPage}
+                            onClick={() => getPage({ offset: nextOffset, limit: [1, 1] })}
+                            className={`button button-action s-retry-button ${isLastPage ? "disabled" : ""}`}
+                        >
+                            Next
+                        </button>
+                    </div>
+                </div>
+            );
+        },
+    },
+    onDemandExecution: {
+        props: {
+            dimensions: [[Model.measure(totalSalesIdentifier)]],
+            autoLoadFirstPage: false,
+        },
+        label: "On Demand Execution",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result, getPage }) => {
+            return (
+                <div
+                    className="s-bucket-executor-kpi"
+                    style={{
+                        display: "flex",
+                        flexDirection: "column",
+                        alignItems: "flex-start",
+                        minHeight: "3em",
+                        lineHeight: "1em",
+                        margin: "10px 0",
+                        fontSize: 30,
+                        verticalAlign: "bottom",
+                        fontWeight: 700,
+                    }}
+                >
+                    {result ? result.data[0] : <p>No data</p>}
+                    <div>
+                        <button
+                            onClick={() => getPage({ offset: [0, 0], limit: [1, 1] })}
+                            className="button button-action s-retry-button"
+                        >
+                            Get Data
+                        </button>
+                    </div>
+                </div>
+            );
+        },
+    },
+};
+
+export class BucketExecutorExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
Nitpick: you don't need a constructor since you do not use the props in there anyway ;)